Source/Charts/Charts/PieChartView.swift

```diff
@@ -71,6 +71,9 @@ open class PieChartView: PieRadarChartViewBase
     
     /// maximum angle for this pie
     fileprivate var _maxAngle: CGFloat = 360.0
+    
+    /// Extra offset value to be added in the radius to move data label outside the chart
+    fileprivate var _dataLabelOffset: CGFloat = 0.0
```

Comment:
maybe we need a more precise word to describe this `offset` as this is eventually a multiplier, not plus

Reply:
Yes i agree